octobot_trading/producers/simulator/recent_trade_updater_simulator.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, channel, importer):
 
         self.last_timestamp_pushed = 0
         self.time_consumer = None
+        # Only generate recent trades from the shortest handled time frame
+        self.recent_trades_time_frame = self.channel.exchange_manager.exchange_config.traded_time_frames[-1].value
```

Comment:
Are you sure this time frame list is sorted ?

Reply:
Yep, there is always at least one timeframe there

Reply:
Yes but are you sure `traded_time_frames[-1]` is the min ?